Sources/GRPCHTTP2Core/Server/HTTP2ServerTransport.swift

```diff
@@ -188,4 +188,68 @@ extension HTTP2ServerTransport.Config {
       Self(maxRequestPayloadSize: 4 * 1024 * 1024)
     }
   }
+
+  public struct TLS: Sendable {
```

Comment:
Didn't we discuss putting this and `TLSConfig` in the posix module as it's specific to the NIOPosix impl.?